.github/CODEOWNERS

```
@@ -3,6 +3,7 @@
 /pages.it/ @mebeim @yutyo
 /pages.pl/ @acuteenvy
 /pages.pt_BR/ @waldyrious
+/pages.pt_BR/ @isaacvicente
```

```suggestion
/pages.pt_BR/ @waldyrious @isaacvicente
```

Similarly, can you add yourself like this in line 17?